sdk/src/wallet/operations/transaction/build_transaction.rs

```diff
@@ -75,9 +85,10 @@ where
         }
 
         // BlockIssuanceCreditContextInput requires a CommitmentContextInput.
-        if context_inputs
+        if (context_inputs
             .iter()
             .any(|c| c.kind() == BlockIssuanceCreditContextInput::KIND)
+            || commitment_context_input_required)
```

Comment:
maybe just set this value here and have a check for it later in the file? I know my PR will prefer that way

Reply:
Set which value?

Reply:
We can also just merge your first and I will adjust

Reply:
sure, I did it in my PR already